pages/index.js

```diff
@@ -233,6 +233,7 @@ export default function HomePage() {
             </ActiveBadge>
           </ActiveFilterRow>
         )}
+        <ImportExportDataInCsv transactions={filteredTransactions} />
```

Comment:
don't you want to place them below the Pagination? The location was great, but they should not be placed inside the pagination component.